packages/common/src/ui/components/buttons/SplitButton.stories.tsx

```diff
@@ -1,28 +1,85 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
```

Comment:
again change here is to support the change to the internal state. Additionally added the disabled showcase and RTL support